### DISCUSSION TITLE
Downgrade from API v3 to API v2

### DISCUSSION THREAD

**Discussion by crmpicco** [etcd-io/etcd#14622] [Q&A]
I have etcd installed through Homebrew on Mac.
```
etcdctl version  
etcdctl version: 3.5.1
API version: 3.5
```
I would like to downgrade the API version to v2, however when I cheekily set the environment variable it then renders the cluster unhealthy.

```
export ETCDCTL_API=2

etcdctl cluster-health
cluster may be unhealthy: failed to list members
Error:  unexpected status code 404
```
It may seem counter-productive to downgrade API version, however I am working with a legacy app and I am not ready to upgrade quite yet.

The cluster in question is only on my local development environment so I have no issue blowing it away if need be, but i'm unsure if that will get me to API v2.

Is there a way to get back to API v2?

**Comment by tjungblu** [comment]
I think you need to just start etcd with `-enable-v2` in 3.5.x.
